src/main/java/org/jenkinsci/plugins/tokenmacro/TokenMacro.java

```diff
@@ -141,7 +144,12 @@ public boolean hasNestedContent() {
      * All registered extension points.
      */
     public static ExtensionList<TokenMacro> all() {
-        return Jenkins.getInstance().getExtensionList(TokenMacro.class);
+        final Jenkins jenkins = Jenkins.getInstance();
+        if (jenkins != null) {
+            return jenkins.getExtensionList(TokenMacro.class);
+        } else {
+            return ExtensionList.create((Jenkins) null, TokenMacro.class);
```

Comment:
is there any point creating a detached list rather than throwing an Exception?
(ie mimic what `Jenkins.getActiveInstance()` does) - will this mask issues in anyone trying to get extensions whilst Jenkins is still initing?